modules/cnf-du-updating-crio.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-crio_{context}"]
+
+= Updating CRI-O
+
+In support of this feature, CRI-O supports new configuration settings. These are applied by the MachineConfig specified above.
+
+----
+[crio.runtime.workloads.{workload-type}]
+  activation_annotation = "target.workload.openshift.io/{workload-type}"
+  annotation_prefix = "resources.workload.openshift.io"
+  resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" }
+----
+
+* Use the `activation_annotation` field to match pods that should be treated as having the workload type. The annotation key on the pod is compared for an exact match against the value specified in the configuration file. In this release, the only supported workload-type is `management`.
+
+* The `annotation_prefix` is the start of the annotation key that passes settings from the admission hook down to CRI-O.
+
+* The `resources` map associates annotation suffixes with default values. CRI-O defines a well-known set of resources and other values are not allowed. The cpuset value must match the kubelet configuration file and the reserved CPU set in the applied PerformanceProfile.
+
+In the management workload case, it is configured as follows:
+----
```

Comment:
* Please add a blank line between the intro para and the code block, in accordance with the doc guidelines for [Code blocks](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).
* Please add "[source,terminal]" for the same reason.
* Consider adding a block title.